### ISSUE TITLE
Seems that a raw iroh node defaults to dialing staging relays

### ISSUE THREAD

**Issue by b5** [n0-computer/iroh#2696]
This is a fairly stock import of iroh into a rust program, and I'm getting "closed" messages & dropped packets. I'm seeing a lot of

```
WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
```

<details>
<pre>
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: Iroh CDN is running on: 0.0.0.0:8080
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: Node ID:                wvwswoepkpklxlogohf3mswpgo5eawqo3bia2zq4e5un34k5tuqa
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: Doc ID:                 pcg32u3wumntsl5l57gzt76ccq4c2tbf4hzwlukrio6xvzoiqlxq
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: Data path:              /root/services/iroh-cdn/data
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: Static assets path:     /root/services/iroh-cdn/dist
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:16.802525Z  INFO backend::handlers: serving static assets from: /root/services/iroh-cdn/dist
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:16.818648Z  INFO ep{me=wvwswoepkpklxlog}:poll_recv:disco_in{node=ahpcrcs6tqh6a7ml src=Udp(72.89.32.33:62094)}:handle_pong{m=Pong { tx_id: TransactionId(0x64193C320332EFE5429C04AB), ping_observed_addr: Udp(5.161.245.36:11205) } src=Udp(72.89.32.33:62094)}: iroh_net::magicsock::node_map::best_addr: selecting new direct path for node addr=72.89.32.33:62094 latency=16.169012ms trust_for=6.499997886s
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:16.841643Z  INFO ep{me=wvwswoepkpklxlog}:magicsock:relay-actor: iroh_net::magicsock::relay_actor: adding connection to relay: https://staging-use1-1.relay.iroh.network./ for PublicKey(ahpcrcs6tqh6a7ml)
Sep 05 01:32:16 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:16.916322Z  INFO connect{me=wvwswoepkpklxlog remote=ahpcrcs6tqh6a7ml alpn="/iroh-gossip/0"}:try_send:get_send_addrs{node=ahpcrcs6tqh6a7ml}: iroh_net::magicsock::node_map::node_state: new connection type typ=direct
Sep 05 01:32:26 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:26.988661Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:27 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:27.235455Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:32 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:32.244764Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:36 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:36.985445Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:37 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:37.250555Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:41 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:41.992004Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:42 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:42.243150Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:46 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:46.990425Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:47 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:47.241082Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:51 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:51.986972Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:52 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:52.253711Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:56 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:56.986307Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"
Sep 05 01:32:57 b5-cdn-gateway iroh-cdn[121827]: 2024-09-05T01:32:57.227051Z  WARN ep{me=wvwswoepkpklxlog}:magicsock:relay-actor:active-relay{url=https://staging-use1-1.relay.iroh.network./}: iroh_net::magicsock::relay_actor: dropping received relay packet: "Closed(..)"

</pre>
</details>

Debug level logs:
[logs.txt](https://github.com/user-attachments/files/16882996/logs.txt)

**Comment by dignifiedquire** [comment]
can you check if by any chance `ENV_FORCE_STAGING_RELAYS` is set in your env?

**Comment by dignifiedquire** [comment]
Also this log is saying the node that is attempting to connect to you uses the staging relay, not your node

**Comment by dignifiedquire** [comment]
Fixed in #2700
